openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -3,44 +3,73 @@
 from openfisca_us.tools.general import *
 
 
-class standard_deduction(Variable):
+class basic_standard_deduction(Variable):
     value_type = float
     entity = TaxUnit
-    label = u"Standard deduction, including for dependents, aged and blind"
+    label = "Basic standard deduction"
     definition_period = YEAR
 
-    def formula(taxunit, period, parameters):
-        # calculate basic standard deduction
-
+    def formula(tax_unit, period, parameters):
         STD = parameters(period).tax.deductions.standard
-        MARS = taxunit("MARS", period)
-        MIDR = taxunit("MIDR", period)
-        MARSType = MARS.possible_values
+        MARS = tax_unit("MARS", period)
+        MIDR = tax_unit("MIDR", period)
         c15100_if_DSI = max_(
-            350 + taxunit("earned", period), STD.amount.dependent
+            350 + tax_unit("earned", period), STD.amount.dependent
         )
         basic_if_DSI = min_(STD.amount.filer[MARS], c15100_if_DSI)
         basic_if_not_DSI = where(MIDR, 0, STD.amount.filer[MARS])
         basic_stded = where(
-            taxunit("DSI", period), basic_if_DSI, basic_if_not_DSI
+            tax_unit("DSI", period), basic_if_DSI, basic_if_not_DSI
         )
+        return basic_stded
 
-        # calculate extra standard deduction for aged and blind
+
+class aged_blind_standard_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Aged and blind standard deduction"
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        STD = parameters(period).tax.deductions.standard
+        MARS = tax_unit("MARS", period)
+        MARSType = MARS.possible_values
         num_extra_stded = (
-            taxunit("blind_head", period) * 1
-            + taxunit("blind_spouse", period) * 1
+            tax_unit("blind_head", period) * 1
+            + tax_unit("blind_spouse", period) * 1
+            + (tax_unit("age_head", period) >= 65) * 1
+            + (
+                (MARS == MARSType.JOINT)
+                & (tax_unit("age_spouse", period) >= 65)
+            )
+            * 1
         )
-        extra_joint_multiplier = where(MARS == MARSType.JOINT, 2, 1)
-        num_extra_stded += (
-            taxunit("age_head", period) >= 65
-        ) * extra_joint_multiplier
         extra_stded = num_extra_stded * STD.amount.aged_or_blind[MARS]
+        return extra_stded
+
+
+class standard_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = u"Standard deduction"
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        # calculate basic standard deduction
```

Comment:
I'm guessing this is copied from taxcalc but could we adopt the PEP8 convention of capitalizing the first word in a comment? https://www.python.org/dev/peps/pep-0008/#comments

Reply:
Yeah that's better, and I've done this myself in other places. I'm guessing we should also add this to Black/another formatter to catch this?

Reply:
Black doesn't check comments, and I think autopep8 only breaks comments into multiple lines and adds a space after `#`. Automatically capitalizing could violate PEP8 guidance:
> The first word should be capitalized, **unless it is an identifier that begins with a lower case letter (never alter the case of identifiers!).**

But autopep8's could be a start, should we run `autopep8` before `black`? I filed https://github.com/UBICenter/openfisca-us/issues/43 to discuss.